src/main/resources/application.yml

```diff
@@ -74,6 +74,11 @@ jobs:
     # Runs at 01:40
     # Running outside business hours to avoid certificate data suddenly disappearing.
     cron: "0 40 1 * * *"
+  remove-aed-final-retention-period-data:
+    name: "RemoveAedFinalRetentionPeriodData"
+    # Runs at 01:30
+    # Running outside business hours to avoid AED data suddenly disappearing.
```

Comment:
`# Running outside business hours to avoid Anonymous elector documents suddenly disappearing.`